interfaces/controllers/HomeController.js

```diff
@@ -0,0 +1,8 @@
+module.exports = class HomeController {
+  openApplication() {
+    return {
+      description: "home",
+      forms: [{ action: "startGameWithWord" }]
```

Comment:
is form a appropriate word.  It feels a little tied to a html implementation

maybe `actions: [{ intent: 'startGameWithWord' }]`

In the same way params are knowledge encoded in the test or the html, this "method" thing is possibly just knowledge encoded in the html

Reply:
Yeah I was definitely thinking of HTML i.e. `<form action="">` but that's probably a bad idea at this level.